packages/node_modules/glimmer-runtime/tests/updating-test.ts

```diff
@@ -120,6 +120,18 @@ test("a simple implementation of a dirtying rerender", function() {
   QUnit.notStrictEqual(root.firstChild.firstChild.firstChild, valueNode, "The text node was not blown away");
 });
 
+test('an if helper should work', function() {
+  let object: any = { condition: true, value: 'hello world' };
+  let template = compile('<div>{{#if condition}}<p>{{value}}</p>{{else}}<p>Nothing</p>{{/if}}</div>');
+  let result = render(template, object);
+
+  equalTokens(root, '<div><p>hello world</p></div>', "Initial render");
+
+  object.condition = [];
+  result.rerender();
+  equalTokens(root, '<div><p>Nothing</p></div>');
+});
+
```

Comment:
Can you rename this to `The if helper should consider an empty array falsy` and add one for the `unless` helper too?

Perhaps it might be better in this shape:

1. Start with condition = `[]`
2. Rerender with no change
3. Rerender with condition.push("stuff");
4. Rerender with condition.pop(); (or `condition = []`?)
```

That way we will have adequate coverage for both the initial render and updating passes